third_party/ml_metadata/metadata_store/metadata_store_go_wrap/metadata_store_go_wrap.go

```diff
@@ -0,0 +1,580 @@
+extern swig_type_6 _wrap_GetArtifactTypesByID_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_7 arg2, uintptr_t arg3);
+extern swig_type_8 _wrap_PutExecutionType_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_9 arg2, uintptr_t arg3);
+extern swig_type_10 _wrap_GetExecutionType_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_11 arg2, uintptr_t arg3);
+extern swig_type_12 _wrap_GetExecutionTypesByID_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_13 arg2, uintptr_t arg3);
+extern swig_type_14 _wrap_PutArtifacts_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_15 arg2, uintptr_t arg3);
+extern swig_type_16 _wrap_PutExecutions_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_17 arg2, uintptr_t arg3);
+extern swig_type_18 _wrap_GetArtifactsByID_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_19 arg2, uintptr_t arg3);
+extern swig_type_20 _wrap_GetExecutionsByID_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_21 arg2, uintptr_t arg3);
+extern swig_type_22 _wrap_PutEvents_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_23 arg2, uintptr_t arg3);
+extern swig_type_24 _wrap_GetEventsByArtifactIDs_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_25 arg2, uintptr_t arg3);
+extern swig_type_26 _wrap_GetEventsByExecutionIDs_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_27 arg2, uintptr_t arg3);
+extern swig_type_28 _wrap_GetArtifacts_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_29 arg2, uintptr_t arg3);
+extern swig_type_30 _wrap_GetArtifactsByType_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_31 arg2, uintptr_t arg3);
+extern swig_type_32 _wrap_GetArtifactsByURI_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_33 arg2, uintptr_t arg3);
+extern swig_type_34 _wrap_GetExecutions_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_35 arg2, uintptr_t arg3);
+extern swig_type_36 _wrap_GetExecutionsByType_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, swig_type_37 arg2, uintptr_t arg3);
+extern uintptr_t _wrap_new_Status__SWIG_0_metadata_store_go_wrap_79669df856d5e5b8(void);
+extern uintptr_t _wrap_new_Status__SWIG_1_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1, uintptr_t arg2);
+extern uintptr_t _wrap_new_Status__SWIG_2_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1);
+extern _Bool _wrap_Status_ok_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1);
+extern uintptr_t _wrap_Status_code_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1);
+extern swig_type_38 _wrap_Status_error_message_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1);
+extern void _wrap_delete_Status_metadata_store_go_wrap_79669df856d5e5b8(uintptr_t arg1);
+#undef intgo
+
+#cgo CXXFLAGS: -std=c++11
+#cgo CXXFLAGS: -I${SRCDIR}/../../../../bazel-metadata/external/com_google_absl
```

Comment:
@zhenghuiwang These are the flags I added to enable `go build` to work with `cgo`.